contrib/pylightning/lightning/lightning.py

```diff
@@ -275,13 +275,14 @@ def getinfo(self):
         """
         return self.call("getinfo")
 
-    def sendpay(self, route, payment_hash):
+    def sendpay(self, route, payment_hash, description=None):
```

Comment:
Maybe default to "" here instead of empty, since it really shouldn't be NULL.  Or we could insist on it and just break backwards compat :(